portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,9 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
-function getHelloArrowFunctions() {
-  fetch('/data').then(response => response.text()).then((quote) => {
+function getHello() {
+  fetch('/data').then(response => response.text())
+  .then((quote) => {
```

Comment:
I'm sorry for missing this before now, but should `quote` be `greeting` or something else?

Reply:
I don't think so?